controlpanel/api/rules.py

```diff
@@ -29,6 +29,18 @@ def is_app_admin(user, obj):
     if obj is None:
         return True
 
+    if is_superuser(user):
+        return True
+
+    if is_superuser(user):
+        return True
+
+    if is_superuser(user):
+        return True
+
+    if is_superuser(user):
+        return True
```

Comment:
This same `if` statement is repeated for some reason, maybe an editor glitch? 🤷‍♂️

Reply:
Huh.... that's weird. Yeah, it's probably vim playing up. ;-) I'll fix an push.

Reply:
OK... fixed. I also rebased with master and force pushed.